projects/core/src/lib/components/edit/state/edit.reducer.ts

```diff
@@ -57,6 +66,67 @@ const onLoadEditFeaturesSuccess = (
   };
 };
 
+const onLoadCopyFeaturesSuccess = (
+  state: EditState,
+  payload: ReturnType<typeof EditActions.loadCopyFeaturesSuccess>,
+): EditState => {
+  state = {
+    ...state,
+    loadStatus: LoadingStateEnum.LOADED,
+  };
+  const geometry = payload.featureInfo[0]?.features[0]?.geometry;
+  if (!geometry) {
+    return state;
+  }
+
+  // Deselect a feature by checking if the geometry WKT is already in the copiedFeatures array (can't search by fid)
+  const sameGeometryIndex = state.copiedFeatures.findIndex(f => f.geometry === geometry);
+  const copiedFeatures = sameGeometryIndex !== -1
+    ? state.copiedFeatures.filter((_, idx) => idx !== sameGeometryIndex)
```

Comment:
most of the times in reducers we use something like `[ ...state.copiedFeatures.slice(0, idx), ...state.copiedFeatures.slice(idx + 1) ]` to remove by index.